File: Beecrowd/funcoes.cpp
//https://judge.beecrowd.com/pt/problems/view/1555
#include <bits/stdc++.h>
using namespace std;

int main(){
    int t; cin >> t;
    while(t--){
        int x, y; cin >> x >> y;

        int r = ((3*x)*(3*x))+(y*y);
        int b = (2*(x*x))+((5*y)*(5*y));
        int c = (-100*x)+(y*y*y);

        if(r > b && r > c){
            cout << "Rafael ganhou" << endl;
        }else if(b > r && b > c){
            cout << "Beto ganhou" << endl;
        }else{
            cout << "Carlos ganhou" << endl;
        }
    }

    return 0;
}

File: OBI/funcoes.py
#https://judge.beecrowd.com/pt/problems/view/1555
n = int(input())

for i in range(n):
    x, y = input().split()
    x = int(x)
    y = int(y)

    r = (3*x)*(3*x) + y*y
    b = 2*x*x + (5*y)*(5*y)
    c = -100*x + y*y*y

    if(r > b and r > c):
        print("Rafael ganhou")
    elif(b > c):
        print("Beto ganhou")
    else:
        print("Carlos ganhou")